packages/feedback/src/sendFeedback.ts

```diff
@@ -6,7 +6,7 @@ import type { SendFeedbackOptions } from './types';
 import { sendFeedbackRequest } from './util/sendFeedbackRequest';
 
 interface SendFeedbackParams {
-  message: string;
+  comments: string;
```

Comment:
I find it a bit weird, API wise, that this is in plural form - sounds like an array to me 😅 but if it helps with compatibility 🤷

Reply:
can we keep both and just deal with either on the backend?